sonic-utilities-tests/aclshow_test.py

```diff
@@ -0,0 +1,194 @@
+import sys
+import os
+from StringIO import StringIO
+import mock
+
+test_path = os.path.dirname(os.path.abspath(__file__))
+modules_path = os.path.dirname(test_path)
+scripts_path = os.path.join(modules_path, "scripts")
+sys.path.insert(0, modules_path)
+
+from imp import load_source
+load_source('aclshow', scripts_path+'/aclshow')
+from aclshow import *
+
+import mock_tables.dbconnector
+
+default_output = ''+ \
+"""RULE NAME     TABLE NAME      PACKETS COUNT    BYTES COUNT
```

Comment:
can you add line breakup and have the table header aligned with the data? this applies to all the below table

Reply:
Line breakup makes assertion to fail, as it compares strings.